hcipy/data/eelt_segment_indices.txt

```diff
@@ -0,0 +1,798 @@
+61
```

Comment:
Is there any way to do this programmatically? I'd imagine you didn't type all this by hand.

Reply:
Yes, I did it with a small function. But I do not have any guarantees at the moment that this function always creates the correct mask. I tweaked the parameters until the aperture matched the documentation. Should we keep it like this for now? I would first need to verify that the function works for any combination of parameters.

Reply:
What do you mean with "any combination of parameters"? There is only one ELT pupil that it needs to generate, right?

Reply:
Yes, but my segment finding algorithm finds the masks based on a relative illumination pattern. And this depends on the sampling density and the chosen power cut off. It will fail for  small number of samples. Because I have the actual pupil now, I can probably create a better subset criterion.

Reply:
Remove the file dependency. It is now based on programmed rules.